knip.ts

```diff
@@ -1,56 +1,35 @@
-import type { KnipConfig } from "knip";
+import type { KnipConfig } from 'knip';
 
 const config: KnipConfig = {
-  ignoreWorkspaces: ["packages/nuxt"],
-  ignoreDependencies: ["cz-conventional-changelog"],
+  ignore: [
+    'tools/artillery/**' // workspace but no package.json
+  ],
+  ignoreWorkspaces: [
+    'apps/agent' // golang app
+  ],
   workspaces: {
-    ".": {
-      entry: "checkly.config.ts",
-    },
-    "apps/dashboard": {
-      entry: ["lib/trpc/index.ts", "trpc.config.ts"],
-    },
-    "apps/api": {
-      entry: ["**/*.test.ts", "src/pkg/testutil/*.ts", "src/worker.ts", "./vitest.*.ts"],
```

Comment:
Good example of entry files Knip figures out itself, none of those are needed :)